java/api/src/main/java/io/ray/api/runtimeenv/RuntimeEnv.java

```diff
@@ -1,38 +1,30 @@
 package io.ray.api.runtimeenv;
 
 import io.ray.api.Ray;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import io.ray.api.exception.RuntimeEnvException;
 
 /** This is an experimental API to let you set runtime environments for your actors. */
 public interface RuntimeEnv {
 
-  String toJsonBytes();
+  void set(String name, Object value);
 
-  public static class Builder {
+  public void setJsonStr(String name, String jsonStr) throws RuntimeEnvException;
```

Comment:
It's better to comment for `setJsonStr` and `getJsonStr` due to they are not straightforward.